src/utils/createMediaStyle/index.js

```diff
@@ -0,0 +1,39 @@
+import { breakpointsToArray } from "../breakpointsToArray";
+import { breakpoints } from "../../breakpoints";
+
+const normalizeCSSValue = value =>
+  typeof value === "number" ? `${value}em` : value || "initial";
+
+const hasValue = value => ![null, undefined, NaN].includes(value);
+
+const replaceValue = (str, value, index) =>
+  str.replace(new RegExp(`\\{${index}\\}`, "gmi"), normalizeCSSValue(value));
+
+export const createMediaStyle = (str, ...props) => {
+  const mediaSizes = Object.values(breakpoints);
+  return props.reduce((css, values, index) => {
+    if (!hasValue(values)) {
+      return css;
+    }
+
+    if (!(values instanceof Array) && typeof values === "object") {
```

Comment:
Given the immediately following `if (Array.isArray(values))` statement, it arguably makes more sense to put this conditional statement after the below array check and just simplify it to `typeof values === "object`.